pkg/traversal/traversal_test.go

```diff
@@ -147,6 +149,8 @@ func TestTraversalBytes(t *testing.T) {
 	for _, tc := range testCases {
```

Comment:
does this also needs the `tc := tc`? maybe in the example bellow also

Reply:
correct; fixed in all places